src/main/java/io/jenkins/plugins/credentials/secretsmanager/AwsSecretSource.java

```diff
@@ -51,18 +49,52 @@
         }
     }
 
-    private AWSSecretsManager createClient() throws SdkClientException {
+    @Override
+    public void init() {
+        super.init();
```

Comment:
No point calling super as it is a NO-OP method.
```suggestion
```

Reply:
made the change